docs/api-reference/deckgl.md

```diff
@@ -75,9 +75,13 @@ This is helpful when rendered objects are difficult to target, for example
 irregularly shaped icons, small moving circles or interaction by touch.
 Default `0`.
 
-##### `pixelRatio` (Number, optional)
+##### `useDevicePixelRatio` (Boolean, optional)
 
-Will use device ratio by default.
+Default value is false.
+
+When true, device's full resolution will be used for rendering.
```

Comment:
I think it would be valuable to add a note here (or in the remarks) that DPR is resampled every frame.